storage_controller/src/scheduler.rs

```diff
@@ -32,6 +32,8 @@ pub(crate) struct SchedulerNode {
     shard_count: usize,
     /// How many shards are currently attached on this node, via their [`crate::tenant_shard::IntentState`].
     attached_shard_count: usize,
+    /// Availability zone id in which the node resides
+    az: String,
```

Comment:
Let's invent a newtype for AZ (even if it's just a plain string in the DB, in memory we can use a nice type)

Reply:
Invented in https://github.com/neondatabase/neon/pull/9083/commits/6f63062dd153a999f53e4cc2e15645c7a2f3abf9